Formula/i/ipatool.rb

```diff
@@ -0,0 +1,22 @@
+class Ipatool < Formula
+  desc "CLI tool for searching and downloading app packages from the iOS App Store"
+  homepage "https://github.com/majd/ipatool"
+  url "https://github.com/majd/ipatool/archive/refs/tags/v2.1.4.tar.gz"
+  sha256 "e0e01c88efb94f35a71f664267c6c9ab0e22932804e0af864a0a5cd8d348dbca"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-X github.com/majd/ipatool/v2/cmd.version=#{version}"), "-o", bin/"ipatool"
+
+    generate_completions_from_executable(bin/"ipatool", "completion")
```

Comment:
I believe this is the default
```suggestion
    generate_completions_from_executable
```

Reply:
Seems that it is not correct.

https://github.com/Homebrew/homebrew-core/actions/runs/11995086748/job/33437987406